modules/kinesis/src/main/scala/com/snowplowanalytics/snowplow/sinks/kinesis/KinesisSinkConfig.scala

```diff
@@ -7,22 +7,31 @@
  */
 package com.snowplowanalytics.snowplow.sinks.kinesis
 
+import cats.Id
 import io.circe._
 import io.circe.generic.semiauto._
 
 import com.snowplowanalytics.snowplow.kinesis.BackoffPolicy
 
 import java.net.URI
 
-case class KinesisSinkConfig(
-  streamName: String,
+case class KinesisSinkConfigM[M[_]](
+  streamName: M[String],
   throttledBackoffPolicy: BackoffPolicy,
   recordLimit: Int,
   byteLimit: Int,
   customEndpoint: Option[URI]
 )
 
-object KinesisSinkConfig {
```

Comment:
Should we keep it and add `apply()` so that we can keep existing builders in other apps than Enrich?

Reply:
I think the builders you're talking about only exist in tests (unless I'm forgetting somewhere?)

In which case no I don't think we should add code that is not needed in main code.

Reply:
Indeed they should only exist in tests.

> I don't think we should add code that is not needed in main code

Fair point!